index.bs

```diff
@@ -872,27 +872,30 @@ Advisory Board (AB)</h4>
 <h5 id=ab-role>
 Role of the Advisory Board</h5>
 
-	The <dfn export lt="Advisory Board|AB">Advisory Board</dfn> provides ongoing guidance to the Team
+	The <dfn export lt="Advisory Board|AB">Advisory Board</dfn>
+	is elected by the Membership to advance the effectiveness of W3C as a community by:
+
+	<ul>
+	<li>Providing ongoing guidance to the Team
 	on issues of strategy,
 	management,
 	legal matters,
 	process,
+	community engagement,
 	and conflict resolution.
-	The Advisory Board also serves the Members
+	<li>Serving the Members
 	by tracking issues raised between Advisory Committee meetings,
 	soliciting Member comments on such issues,
 	and proposing actions to resolve these issues.
-	The Advisory Board manages the <a href="#GAProcess">evolution of the Process Document</a>.
-	As part of a [=W3C Council=],
-	members of the [=Advisory Board=] hear and adjudicate on [=Submission Appeals=]
-	and [=Formal Objections=].
+	<li>Managing the <a href="#GAProcess">evolution of the Process Document</a>.
+	</ul>
 
 	The [=Advisory Board=] is distinct from the [=Board of Directors=]
-	and has no decision-making authority within W3C;
+	and has no decision-making authority over W3C;
 	its role is strictly advisory.
```

Comment:
```suggestion
```
Given the AB's role in managing the Process document (and now also The Vision, and facilitating the Code of Conduct, which can be a separate PR), and FO Councils, it's high time to remove this legacy phrase which causes more confusion than clarity in practice.

Reply:
I am cautiously in favor of this proposed removal. Removing this sentence does not cause the AB to gain any decision power it does not otherwise have, so this seems to be a safe change, and "strictly" hasn't really been true, so this is a likely source of confusion.

Reply:
In fact, I don't believe that sentence is correct anymore. The Process says: "Revision of the W3C Process and related documents (see below) undergoes similar [consensus](https://www.w3.org/policies/process/20250818/#def-Consensus)-building processes as for [technical reports](https://www.w3.org/policies/process/20250818/#technical-report), with the [Advisory Board](https://www.w3.org/policies/process/20250818/#advisory-board) acting as the sponsoring [Working Group](https://www.w3.org/policies/process/20250818/#GroupsWG).".

So, the AB does have decision powers, similar to Working Groups. Both are subject to AC Reviews.